{"uuid": "e7ab9ee6-80dd-4340-b4d9-b64530a95f9e", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "6004eca2-40e6-4481-9e20-7d5d0de67546", "event": "runner_on_unreachable", "pid": 8240, "created": "2022-04-22T13:39:00.783304", "parent_uuid": "9e73ed77-7fb2-7cad-1091-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "ef3a297e-c281-4217-b9e4-fa8665106171", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-7cad-1091-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-7cad-1091-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:39:00.763486", "end": "2022-04-22T13:39:00.783203", "duration": 0.019717, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "e7ab9ee6-80dd-4340-b4d9-b64530a95f9e"}}